SPECS/bcc.spec

```diff
@@ -10,42 +10,37 @@ License:        ASL 2.0
 URL:            https://github.com/iovisor/bcc
 Source0:        bcc.tar.gz
 
-BuildArch:      x86_64
-BuildRequires:  bison, cmake >= 2.8.7, flex, gcc, gcc-c++, python2-devel, elfutils-libelf-devel-static
+ExclusiveArch: x86_64
+BuildRequires: bison cmake >= 2.8.7 flex
+BuildRequires: gcc gcc-c++ python2-devel elfutils-libelf-devel-static
+BuildRequires: luajit luajit-devel
+BuildRequires: llvm-devel llvm-static
+BuildRequires: clang-devel
```

Comment:
As part of the rpmbuild process the executables are scanned for shared libraries requirements. This is can be seen toward the end of https://kojipkgs.fedoraproject.org//work/tasks/4695/16554695/build.log  .  rpmlint will complain if there are explicit dependencies on the rpms that contain those libraries.